sklearn/svm/tests/test_svm.py

```diff
@@ -852,3 +854,19 @@ def test_hasattr_predict_proba():
     assert_true(hasattr(G, 'predict_proba'))
     msg = "predict_proba is not available when fitted with probability=False"
     assert_raise_message(NotFittedError, msg, G.predict_proba, iris.data)
+
+
+def test_decision_function_shape_svc():
+    X, y = make_blobs(random_state=0)
+    for two_class in [True, False]:
+        X_ = X
+        y_ = y
+        if two_class:
+            msk = np.logical_or(y_ == 0, y_ == 1)
```

Comment:
or iterate over ``n_classes in [2, 3]`` and give that to ``make_blobs(centers=n_classes)`` (I think that works)